### ISSUE TITLE
LIT4120Targwama

### ISSUE THREAD

**Issue by DenisNosnitsin1970** [BetaMasaheft/Manuscripts#704]
LIT4120Targwama used in Eth 157 seems to be a doublet for LIT5549TribesApostles that has been used in multiple descriptions for a short text on the origin countries of the Apostles. I would delete the latter and use only the former. Now, there is a problem: the text exists in at least two versions, longer and shorter, but not all manuscripts are available for check so that we would be able to assign the witnesses. Eth 157 is not available. I would still delete LIT4120Targwama and use LIT5549TribesApostles only as one general record, without further specification. Later someone can try to classify the witnesses. Do you have any objections?

**Comment by eu-genia** [comment]
In PR https://github.com/BetaMasaheft/Works/pull/268 you seem to have indicated that LIT5849GenealogyApostles is (also) a doublet for LIT5549TribesApostles - or you mean that these are doublets for sure and only LIT4120Targwama is doubtful?
(issues should be created under issues, not manuscripts, but I don't know how to move it now that it is created)

**Comment by DenisNosnitsin1970** [comment]
Yes, there are have been two doublets for this work. The work was poorly visible. I want to make LIT5549TribesApostles the only one.

**Comment by eu-genia** [comment]
OK for me as long as you follow the guidelines for deleting doublets and clean all relevant records. (see my comment to the PR)
https://betamasaheft.eu/Guidelines/?q=delete&id=create
